contracts/feature-tests/basic-features/src/small_num_overflow_test_ops.rs

```diff
@@ -0,0 +1,115 @@
+multiversx_sc::imports!();
+
+/// Used for testing overflow on small int types
+#[multiversx_sc::module]
+#[allow(clippy::redundant_clone)]
+pub trait SmallIntOverflow {
+    #[endpoint]
+    #[allow(arithmetic_overflow)]
+    fn usize(&self) -> usize {
```

Comment:
could we rename them to something like `overflow_usize`, `overflow_*`
just to avoid amiguity in Rust, these are also type names